roles/openshift_cli/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- openshift_facts:
+    role: common
+    local_facts:
+      deployment_type: "{{ openshift_deployment_type }}"
+      
+- name: Install clients
+  yum: pkg={{ openshift.common.service_type }}-clients state=installed
+  when: not openshift.common.is_atomic | bool
```

Comment:
I guess technically we could install the client outside of the container when the host isn't atomic...  I'm not sure that we should though.